### PR TITLE
add package info

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
reaction to #358 

It would be fine to make a homepage for example from the main README file, as Github offers
see: https://guides.github.com/features/pages/
it is about %min and couple clicks... :)

### CODE REVIEW

File: pytorch_lightning/__init__.py [hunk 1]
```diff
@@ -7,3 +7,13 @@
     'LightningModule',
     'data_loader',
 ]
+
+__version__ = '0.5.2'
+__author__ = ' William Falcon at al.'
```
Comment (Ir1d) [inline pytorch_lightning/__init__.py]:
```suggestion
__author__ = ' William Falcon et al.'
```

**Comment by Borda** [comment]
btw, why is the package name `pytorch_lightning` not just `torch_lightning` the same way as PyTorch repo but imported as `import torch`

**Comment by Borda** [comment]
seems to he be failing on unrelated code...

**Comment by williamFalcon** [comment]
fixed tests. can you rebase master?

**Comment by williamFalcon** [comment]
@Borda I had to revert this because it broke the install.

```
(pl3) [user@machine5 pytorch-lightning]$ pip install numpy
Requirement already satisfied: numpy in /misc/scratch1/_Group/falcon/software/miniconda3/envs/pl3/lib/python3.6/site-packages (1.16.4)
(pl3) [user@machine5 pytorch-lightning]$ pip install -e .
Obtaining file:///home/user/Developer/pytorch-lightning
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /home/user/media/falcon_scratch1/software/miniconda3/envs/pl3/bin/python /home/user/media/falcon_scratch1/software/miniconda3/envs/pl3/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpw4qf53yy
       cwd: /home/user/Developer/pytorch-lightning
  Complete output (22 lines):
  Traceback (most recent call last):
    File "/home/user/media/falcon_scratch1/software/miniconda3/envs/pl3/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 207, in <module>
      main()
    File "/home/user/media/falcon_scratch1/software/miniconda3/envs/pl3/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 197, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/home/user/media/falcon_scratch1/software/miniconda3/envs/pl3/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 54, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-30cwdsvd/overlay/lib/python3.6/site-packages/setuptools/build_meta.py", line 146, in get_requires_for_build_wheel
      return self._get_build_requires(config_settings, requirements=['wheel'])
    File "/tmp/pip-build-env-30cwdsvd/overlay/lib/python3.6/site-packages/setuptools/build_meta.py", line 127, in _get_build_requires
      self.run_setup()
    File "/tmp/pip-build-env-30cwdsvd/overlay/lib/python3.6/site-packages/setuptools/build_meta.py", line 237, in run_setup
      self).run_setup(setup_script=setup_script)
    File "/tmp/pip-build-env-30cwdsvd/overlay/lib/python3.6/site-packages/setuptools/build_meta.py", line 142, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 10, in <module>
      import pytorch_lightning
    File "/home/user/Developer/pytorch-lightning/pytorch_lightning/__init__.py", line 1, in <module>
      from .trainer.trainer import Trainer
    File "/home/user/Developer/pytorch-lightning/pytorch_lightning/trainer/trainer.py", line 9, in <module>
      import numpy as np
  ModuleNotFoundError: No module named 'numpy'
```

**Comment by williamFalcon** [comment]
Please fix, resubmit PR and test (inluding pip install -e .)

**Pull request merged**